libs/components/indicators/src/lib/modules/tokens/token.component.ts

```diff
@@ -86,40 +80,31 @@ export class SkyTokenComponent {
   @Output()
   public tokenFocus = new EventEmitter<void>();
 
-  /**
-   * @internal
-   */
-  public get tabIndex(): number | boolean {
-    return this.focusable ? 0 : -1;
-  }
+  public tokenActive: boolean | undefined;
+  public closeActive: boolean | undefined;
```

Comment:
Since `undefined` is falsey, could we set these deliberately to `false` and avoid using `| undefined`?

Reply:
Fixed.